code/game/machinery/vending.dm

```diff
@@ -1118,17 +1118,30 @@ var/global/num_vending_terminals = 1
 	name = "PTech"
 	desc = "Cartridges for PDAs"
 	product_slogans = "Carts to go!"
+	req_access_txt = "15"
```

Comment:
It's fine if you just fix your own contributions. Shameful that the other vends do it, though.